assets/composer/settings.local.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+if (getenv('OPENID_CONNECT_PARAMS') == true) {
+  $openid_config_params = json_decode(getenv('OPENID_CONNECT_PARAMS'), true);
+  $config['openid_connect.settings']['always_save_userinfo'] = $openid_config_params['always_save_userinfo'];
+  $config['openid_connect.settings']['connect_existing_users'] = $openid_config_params['connect_existing_users'];
+  $config['openid_connect.settings']['override_registration_settings'] = $openid_config_params['override_registration_settings'];
+  $config['openid_connect.settings']['end_session_enabled'] = $openid_config_params['end_session_enabled'];
+  $config['openid_connect.settings']['user_login_display'] = $openid_config_params['user_login_display'];
+  foreach($openid_config_params['clients'] as $client => $value) {
+    $config['openid_connect.client.' . $client]['status'] = $value['status'];
```

Comment:
Can we just set the whole client in from the environment variable, not try and put some in the config and other bits in the environment?  We don't have flow or profiles in this, which could cause issues, and we are forcing two configs to be present, but we don't necessarily want those two in each site.

Reply:
I did put some effort into do that and removing the config, but so far it's a no go.